crates/dbt-common/src/tracing/formatters/duration.rs

```diff
@@ -76,3 +81,21 @@ pub fn format_timestamp_utc_zulu(ts: SystemTime) -> String {
     let datetime: DateTime<Utc> = ts.into();
     datetime.to_rfc3339_opts(chrono::SecondsFormat::Micros, true)
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+     #[test]
+    fn test_format_duration_fixed_width() {
+        use std::time::Duration;
+        assert_eq!(format_duration_fixed_width(Duration::from_secs(90061)), "   >24h");
```

Comment:
@mishamsk You think it's okay to have these hard coded use cases? or is that too inelastic?